install/linux/usr/share/odemis/sim/cryosecom-sim.yaml

```diff
@@ -164,7 +164,7 @@ CryoSECOM: {
   class: tmcm.TMCLController,
   init: {
     port: "/dev/fake6",
-    ustepsize: [5.9e-9, 5.9e-9, 5.9e-9, 5.9e-9, 5.9e-9],
+    ustepsize: [5.9e-9, 5.9e-9, 5.9e-9, 5.9e-6, 5.9e-6],
```

Comment:
Had to lower ustepsize for `rx` and `rx`, otherwise simulator times out before getting to coating `rx` angel = 0.4363